eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -101,16 +101,19 @@
   <suppress checks="com.azure.tools.checkstyle.checks.ServiceClientBuilderCheck" files=".*[/\\]src[/\\]test[/\\]java[/\\].*\.java"/>
```

Comment:
@mssfang Did you have a PR out that converted these to use a regex so we don't have these sort of duplicate lines?

Reply:
It would be good to use a regex for samples, too if we're not going to apply the rules there.

Reply:
Created a PR that includes the conversion. #5032